### PR TITLE
polkadot without any fee subtraction for max button

### PR DESCRIPTION
**Pull request by enriquesouza**
This should resolve the issue https://github.com/vultisig/vultisig-ios/issues/1390

We need to alert the user they must leave a few fractions to pay the fees

### CODE REVIEW

File: VultisigApp/VultisigApp/View Models/SendCryptoViewModel.swift [hunk 1]
```diff
@@ -157,6 +157,17 @@ class SendCryptoViewModel: ObservableObject, TransferViewModel {
                 
                 await convertToFiat(newValue: tx.amount, tx: tx)
                 
+                isLoading = false
+            }
+        case .polkadot:
+            Task {
+                await BalanceService.shared.updateBalance(for: tx.coin)
+                var gas = BigInt.zero
```
Comment (flypaper0) [inline VultisigApp/VultisigApp/View Models/SendCryptoViewModel.swift:165]: do we need that var? 

why we're passing zero getMaxValue?

Comment (johnnyluo) [reply VultisigApp/VultisigApp/View Models/SendCryptoViewModel.swift:165]: do you mind to get it fixed , so we can get this one merged?

Comment (enriquesouza) [reply VultisigApp/VultisigApp/View Models/SendCryptoViewModel.swift:165]: Sure, sorry for the delay.

**Review by johnnyluo — APPROVED** [review]
LGTM

**Pull request merged**